src/Core/tests/DeviceTests/Handlers/Image/ImageHandlerTests.cs

```diff
@@ -111,8 +111,8 @@ public async Task SourceInitializesCorrectly(string filename, string colorHex)
 		}
 
 		[Theory(
-#if __IOS__ || __ANDROID__
-			Skip = "Animated GIFs are not yet supported on iOS. Test failing on ANDROID"
+#if __IOS__ || __ANDROID_30__
+			Skip = "Animated GIFs are not yet supported on iOS."
```

Comment:
Is this 30+ intentional?

So 21/29 is working and 30/33 is failing? Do we have any idea why?